release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -293,6 +307,20 @@ You can now set the maximum length of the syslog message in the Ingress Controll
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters_configuring-ingress[Ingress Controller configuration parameters].
 
+[id="ocp-4-10-openstack-dcn-for-enterprise"]
+==== Support for {rh-openstack} DCN (Technology Preview)
+
+You can now deploy a cluster on a {rh-openstack-first} deployment that uses a link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/distributed_compute_node_and_storage_deployment/index[distributed compute node (DCN)] configuration. There are a number of limitations:
+
+* Only {rh-openstack} version 16 is supported.
+* For {rh-openstack} 16.1.4, only hyper-converged infrastructure (HCI) and Ceph technologies are supported at the edge.
+* For {rh-openstack} 16.2, non-HCI and Ceph technologies are supported as well.
+* Networks must be created ahead of time (Bring Your Own Network) as either tenant or provider networks. These networks must be scheduled in the appropriate availability zones.
+
+[id="ocp-4-10-external-cloud-providers-openstack"]
+==== Support for external cloud providers for clusters on {rh-openstack} (Technology Preview)
+
+Clusters that run on {rh-openstack} now support external cloud providers. 
```

Comment:
Do we need to mention it's behind the TechPreviewNoUpgrade feature flag?

Reply:
```suggestion
Clusters that run on {rh-openstack} now support external cloud providers. This capability is available as part of the `TechPreviewNoUpgrade` feature set.

// TODO: Link 
```